recipes/rsatoolbox/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "rsatoolbox" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rsatoolbox-{{ version }}.tar.gz
+  sha256: 2d091cbaa33373bf9da4df5ca8d127f0e427431a3db726076090ab2d54fe1213
+
+build:
+  skip: true  # [py<37 or py>310 or py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - cython # >=3.0.0a11
```

Comment:
If you absolutely want/need cython 3, you can add a `conda_build_config.yaml` file like [this one](https://github.com/conda-forge/aimrocks-feedstock/blob/main/recipe/conda_build_config.yaml).

Reply:
Turned out this was just because we wanted to design against Cython3, but it also builds on the latest Cython 2.9